dbcon/mysql/ha_mcs_impl.cpp

```diff
@@ -3185,7 +3218,7 @@ void ha_mcs_impl_start_bulk_insert(ha_rows rows, TABLE* table, bool is_cache_ins
 
             try
             {
-                colrids = csc->columnRIDs(tableName);
+                colrids = csc->columnRIDs(tableName, lower_case_table_names);
```

Comment:
This should have `lower_case_table_names` as the second argument.

Reply:
false added as second arg